src/Master.cpp

```diff
@@ -152,7 +152,7 @@ void Master::statistics() {
     Status["files"][FilewriterTaskID] = FilewriterTaskStatus;
   }
   auto Buffer = Status.dump();
```

Comment:
might be best to use std::string here and be explicit,  like the other `dump()` call in logger.cpp:125

Reply:
Thanks, made the change in another branch.